.github/workflows/auto-approve-bot-prs.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Auto-Approve Bot PRs
+
+# This workflow automatically adds labels and approves PRs that match specific criteria:
+# - Created by rhdh-bot (via RHDH GitHub App)
+# - Branch name matches specific patterns (base image updates, version bumps, etc.)
+# - Adds lgtm and approved labels if not present
+
+on:
+  pull_request:
+    types: [opened, reopened, labeled, ready_for_review]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  auto-approve:
+    name: Auto-Approve and Label PRs
+    runs-on: ubuntu-latest
+    
+    # Only run if PR is from rhdh-bot
+    if: github.event.pull_request.user.login == 'rhdh-bot[bot]'
+    
+    steps:
+      - name: Check PR eligibility
+        id: check-eligibility
+        run: |
+          PR_BRANCH="${{ github.event.pull_request.head.ref }}"
+          PR_DRAFT="${{ github.event.pull_request.draft }}"
+          
+          # Don't auto-approve draft PRs
+          if [[ "$PR_DRAFT" == "true" ]]; then
+            echo "eligible=false" >> $GITHUB_OUTPUT
+            echo "reason=PR is in draft state" >> $GITHUB_OUTPUT
+            exit 0
+          fi
+          
+          # Labels will be added automatically if eligible
+          
+          # Define branch patterns that are eligible for auto-approval
+          # Add more patterns as needed
+          ELIGIBLE_PATTERNS=(
+            "^update-base-images-.*"              # Base image updates
```

Comment:
we could change this to start with `chore:` so the semantic comment checker doesn't fail